### ISSUE TITLE
New class training and data

### ISSUE THREAD

**Issue by xaerincl** [GantMan/nsfw_model#37]
Hello!

Im a ML practitioner from Chile (so im sorry in advance for any english mistakes). I was thinking about improving this model with a new NSFW class: violence (gore for example). I understand basic concepts of transfer learning and how to do it, but since you are the expert I would like to know if you can help me with:

1.- How much data of the original classes would you use for this new training? Should I use the 60 gbs or that would be overkill since the model already can already classify the 5 original classes.
2.- How much data do you think i need for the new class?

Btw im so grateful for your code, especially the mobilenet version! how long did it take to train that model?

Cya!

**Comment by xaerincl** [comment]
Ohh and also, i would love to see if you can share your test data so I can check how the model change in accuracy with the new class.

**Comment by jessetrana** [comment]
(There's a bit tucked away in the ["Extra Info" section on the main page](https://github.com/GantMan/nsfw_model#extra-infol) about the dataset.)

**Comment by GantMan** [comment]
Hiya!  I trained the model for a week, on my single GPU 1060

I'm not sure without experimenting how much data you'll need to bring in a new class.  I'd love to hear how your training is going.

As for the training data, please see if it's still hosted here:  https://github.com/GantMan/nsfw_model/issues/11

**Comment by xaerincl** [comment]
A week for the inception model or for the mobilenet one?


btw your link works! any idea how can i download it using ubuntu terminal? im using paperspace to run my models.

**Comment by xaerincl** [comment]
@GantMan btw the training data provided in your link is the same one you used to get 93% accuracy with the inceptionv3 net?

**Comment by GantMan** [comment]
@xaerincl yes!

**Comment by xaerincl** [comment]
I just archieved 91% accuracy on the validation set with a simple mobilenet v2 (with no more layers than the original, i just changed the dimension of the last fc layer from 1000 to 5) in a full day of training with  11 complete epochs! (11cycles through all the data)


Whats was your final accuracy with the test set?

**Comment by camhart** [comment]
@xaerincl are you planning on sharing your model?
